### ISSUE TITLE
add platform_username

### ISSUE THREAD

**Issue by FluentCoding** [meister03/discord-linked-roles#2]
As of right now, setUserMetaData only sends a given [platform_name](https://github.com/meister03/discord-linked-roles/blob/master/src/Core/Application.ts#L71-L72) but I think it would be nice if we could add an optional platform_username as well.
(see [Update Application Role Connection](https://discord.com/developers/docs/resources/user#update-user-application-role-connection))

**Comment by meister03** [comment]
You can set "..." , by not setting the username on the function?

**Comment by meister03** [comment]
The .setUserMetadata has a platform name param...

**Comment by FluentCoding** [comment]
yeah, my request however was to also add platform_username to the function which is different from platform_name, see the official docs that I've referenced in my original post of this issue.

**Comment by meister03** [comment]
will add the option soon
